aten/src/ATen/cpu/vec/vec256/vec256_qint.h

```diff
@@ -109,8 +109,8 @@ inline Vectorized<float> convert_uint8_to_float(at::vec::Vectorized<uint8_t> src
 }
 
 inline Vectorized<uint8_t> convert_float_to_uint8(at::vec::Vectorized<float> src) {
-  // Convert from float32 to int32
-  __m256i x_values_int32 = _mm256_cvtps_epi32(src);
+  // Convert from float32 to int32, directly trunc
```

Comment:
the wording should be `Convert from float32 to int32 with truncation`

Reply:
Thanks for the suggestion and changed.